### ISSUE TITLE
MultiMC Crashes for my friend, but not for me with same file.

### ISSUE THREAD

**Issue by Lingay6969** [MultiMC/MultiMC5#3276]
System Information
-----------------------------
MultiMC version: 0.6.11

Operating System: Mac OS Catalina 10.15.5

Summary of the issue or suggestion:
----------------------------------------------
I have a version of Valhelsia running on my computer and i can play it perfectly fine. My friend however with the exact same specs and instance can't load in. It crashes as soon as its done loading with Error 255

What should happen:
------------------------------
The game should be playable

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1.

2.

3.

Suspected cause:
---------------------------

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)

https://paste.ee/p/X0XmU

Additional Info:
---------------------------
Thanks much! <3

**Comment by Janrupf** [comment]
Immersive engineering is crashing the game, try updating the mod if possible.

**Comment by Lingay6969** [comment]
This is a complete 1.15.2 modpack that two of us are already using successfully and have a server on. Upgrading immersive engineering would take it past 1.15.2

We are trying to find out why it would not be working on the third friends computer. I made an EXACT copy of my instance and had them use that.

**Comment by Janrupf** [comment]
Could technically be everything. Running the same files on another computer doesn't mean that they behave the same, sadly. There is nothing really what can be done other than maybe bugging the Immersive engineering developer about it, or trying an older version possibly.

**Comment by Lingay6969** [comment]
Previously my friend was trying to run the instance without having java installed but the error message we got was (The Midnight has encountered an error during the Common_Setup Event phase)

In the log there were no errors involving Immersive Engineering. So I don't understand why that is now the problem after installing Java 8.

**Comment by JakeMoe** [comment]
Why would you copy the instance?  Just have the friend import the pack's zipfile directly, and use that.  Just copying the folder from multimc/instances would only get you the files there.  If I remember correctly, there are some mods (Astral Sorcery is one that springs to mind) that save info in the user's .minecraft folder, and it might cause problems if it's not there.

One way or another, I'd have the friend just try a straight import from zip first, and see if it still has problems.

You didn't mention what version of Valhelsia you're running, but upgrading the pack (all three clients, and the server) might be an option as well.

**Comment by Lingay6969** [comment]
The reason we copied the whole instance was because the zip file I originally used had an issue with TheMidnight mod. This was before we realized java was the issue. Also because my friend isn't very good at computers,we wanted to make sure she had the correct optimization settings. 

This did work!
We found a more recent version of immersive engineering 1.15.2 and now she can at least load in. But now that she can get into the server, her computer freezes after a minute or so. Still working out the kinks. She may just not have a good enough computer unfortunately:(
